src/components/ShortAnswer.tsx

```
@@ -0,0 +1,23 @@
+import { color, typography } from "wowds-tokens";
+import BaseAnswer, { type BaseAnswerProps } from "./BaseAnswer";
+
+interface ShortAnswerProps extends BaseAnswerProps {}
```

ESLint

[error] An interface declaring no members is equivalent to its supertype.

(@typescript-eslint/no-empty-object-type)